ci/docs/rosa_cluster_provision.md

```diff
@@ -0,0 +1,34 @@
+# Red Hat OpenShift Service on AWS (ROSA) Cluster Provisioning
+
+`rosa` is a command line tool that simplifies the use of Red Hat OpenShift Service on AWS, also known as ROSA. This cluster is required to manage Hypershift operator for pipeline-service.
```

Comment:
The two sentences are disjointed. The second sentence starts with `This cluster`, the cluster has not been referenced before, only the ROSA service.